### PR TITLE
first pass

### PR DESCRIPTION
**Pull request by MarkhamShofner**
In module 2:

1) The block says that `npm test` should fail at one point, but all my tests were passing then.
2) This line isn't clear to me when it's added. It's the only difference between the js I built off of your instructions and the js you have at the end.
SEARCH_BUTTON: 'input[value="Google Search"]',
3) It seems like we end in the middle of the module? Without running the tests again? The tests are also failing at this point for me when I actually run them.

### CODE REVIEW

File: 2_google_test/README.md [hunk 1]
```diff
@@ -12,7 +29,7 @@ describe('Esri DevSummit Google Search', () => {
     });
 });
 ```
-- Now let's define some constants.
+- Now let's define some constants in the `it()` bock.
```
Comment (drewctate) [inline 2_google_test/README.md]: *block

File: 2_google_test/README.md [hunk 2]
```diff
@@ -28,16 +45,16 @@ const searchText = 'esri devsummit workshops';
 browser.url(url);
 ```
 
-- Time to run the project and see where we're at.
+Time to run the project and see where we're at.
   - `npm i`
   - add a convenience script into `package.json`
   ```json
-  scripts: {
+  "scripts": {
```
Comment (drewctate) [inline 2_google_test/README.md]: good catch

**Comment by drewctate** [comment]
Great @MarkhamShofner. Just need to address a typo

**Comment by drewctate** [comment]
> The block says that npm test should fail at one point, but all my tests were passing then.

The tests don't report a _test_ failure, but they fail to enter the search text into the search bar, so that should be clarified. Back to the whole "these are just my notes" mentality.

> This line isn't clear to me when it's added. It's the only difference between the js I built off of your instructions and the js you have at the end.
SEARCH_BUTTON: 'input[value="Google Search"]',

What you did looks good.

> It seems like we end in the middle of the module? Without running the tests again? The tests are also failing at this point for me when I actually run them.

Yeah, it's probably a good idea to put in a final run-the-tests-and-taste-the-glory step. What were they failing on for you?

**Comment by MarkhamShofner** [comment]
@drewctate failure below

 "spec" Reporter:
------------------------------------------------------------------
[firefox 18.7.0 mac #0-0] Spec: /Users/mark8604/dev/playground/e2ewalkwalk/2_google_test/test/specs/google-search.js
[firefox 18.7.0 mac #0-0] Running: firefox (v18.7.0) on mac
[firefox 18.7.0 mac #0-0] Session ID: 5b772a36-c9d3-4a4e-a235-ad516bf8ae24
[firefox 18.7.0 mac #0-0]
[firefox 18.7.0 mac #0-0] Esri DevSummit Google Search
[firefox 18.7.0 mac #0-0]    ✖ should display correct titles for the first three listings
[firefox 18.7.0 mac #0-0]
[firefox 18.7.0 mac #0-0] 1 failing (2.9s)
[firefox 18.7.0 mac #0-0]
[firefox 18.7.0 mac #0-0] 1) Esri DevSummit Google Search should display correct titles for the first three listings
[firefox 18.7.0 mac #0-0] selector needs to be typeof `string` or `function`

**Comment by drewctate** [comment]
@MarkhamShofner I think the key may be `selector needs to be typeof string or function`. Any way I could take a look at your code?

**Comment by MarkhamShofner** [comment]
@drewctate sure thing! Here's the code as I have it https://github.com/MarkhamShofner/e2e-walkthrough

**Pull request merged**